### PR TITLE
docs: add Nuxt 3 info

### PR DESCRIPTION
**Pull request by frederickfogerty**
This PR adds info about how to install this library with Nuxt v3

### CODE REVIEW

**Comment by frederickfogerty** [comment]
Current dependencies on/for this PR:
* next
  * **PR #439** <a href="https://app.graphite.dev/github/pr/imgix/vue-imgix/439" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/vue-imgix/439?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -114,13 +114,35 @@ app.mount('#app');
...
-However, Nuxt.js now has a built-in [Image component](https://image.nuxtjs.org/getting-started/installation) that can be used alongside imgix.
+#### With `nuxt-image`
+
+Nuxt.js now has a built-in [Image component](https://image.nuxtjs.org/getting-started/installation) that can be used alongside imgix.
 
 You can read more about our `nuxt-image` provider and how to use it in the [Nuxt docs](https://image.nuxtjs.org/providers/imgix).
 
+#### Using this library directly
+
+> For Nuxt v2 support, please look at the [`main`](https://github.com/imgix/vue-imgix/tree/main) branch.
```
Comment (sherwinski) [inline README.md:127]: Was this meant to refer to a different branch?

**Review by sherwinski — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (frederickfogerty) [reply README.md:127]: I don't think so...

Comment (sherwinski) [reply README.md:127]: I thought nuxt2 maps to vue2 and nuxt3 to vue3, so it sounded like nuxt2 support would be found on a different branch or this line should actually say nuxt3?

Comment (frederickfogerty) [reply README.md:127]: Right - this PR was made before we made the decision to switch the branches. In the other PRs that switch the branches this is updated

**Pull request merged**